omod/src/main/webapp/fragments/field/locations.gsp

```diff
@@ -0,0 +1,22 @@
+<%
```

Comment:
Where are you using the fragment? Can you show me the actual code where you use it?

Reply:
You can use this App Definition with this fragment to add the location selector to the patient dashboard. 
See here,
https://gist.github.com/suthagar23/f0b92381aec5d0ad6d70e4eb40eb308d#file-appdefinition-L91

Reply:
Can you create a wiki page for the owa and then put this definition there?

Reply:
Sure, I will create it and post it there with this definition

Reply:
Can you post the url here when done? Having it in place is part of the requirements before merging this.

Reply:
Please have a look at here  @dkayiwa 
https://wiki.openmrs.org/display/docs/Adding+Location+Selection+to+the+Patient+Registration+App